content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,52 @@
+# Machine Learning Model Registries
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Models can be discovered, audited, shared,
+tested, and deployed from here. DVC enables these features on the Git ecosystem,
+reducing friction between development and operations. Other tools from
+[Iterative](https://iterative.ai/) provide more advanced features and better
+usability.
```

Comment:
> tools from Iterative

💡  It would be nice to have a landing page for the list of Iterative.ai products. Right now it's a menu you have to click on in the home page:

![image](https://user-images.githubusercontent.com/1477535/162360673-1af8b29b-9f06-4e68-9994-813fb2a8c409.png)

Reply:
Idea moved to https://github.com/iterative/iterative.ai/issues/431